### ISSUE TITLE
Instructions for Coinbase needs (yet more) clarification

### ISSUE THREAD

**Issue by HughGlaser** [keybase/keybase-issues#1298]
The next screen after the username capture says:
Please post the below text on Coinbase's Keybase page, exactly as it appears.
I couldn't find a "Keybase page" - I spent a while looking, and eventually found there was a "EDIT PUBLIC KEY" link on my public page when logged in.
So some more clarification/correction of the text would be great.

**Comment by marcostudios** [comment]
Can confirm https://www.coinbase.com/marcostudios/public-key doesn't exist

**Comment by malgorithms** [comment]
@marcostudios - with coinbase, you need to activate your "payments page" on coinbase for this to work. If you're doing this from the Keybase website, there's a note about it with a link at the beginning of the dialog process. Sorry it's not clearer ! Will work on that.

**Comment by marcostudios** [comment]
Oh, now it works, thanks :)

Il giorno gio 2 apr 2015 alle ore 14:39 Chris Coyne <
notifications@github.com> ha scritto:

**Comment by jimscarver** [comment]
I cannot find any keybase to post anything on.  I cannot find a "payments" page either.  I see what I received, bought and sold in my wallet.  The link takes me to public key where I posted my public key  Does the text go in with my public key?  HELP.

**Comment by AKRFranko** [comment]
I managed to configure my coinbase account (which I created whilst setting up my keybase.io account) by clicking on the coinbase popup link and pasting in whatever box was there. I got lucky, but it was a very frightening experience, not being familiar with coinbase at all, I couldn't see a relation to the coinbase page I was seeing versus what was worded in the keybase.io popup.  

Potential solutions OTTOMH:

- Better wording. (May not work for all languages in the long run)
- In the keybase popup, provide a "screenshot" of what the 3rd party popup/page looks like.
- Display the full url instead of "coinbase popup" as the text for the popup link.

**Comment by jimscarver** [comment]
Thanks for the reply.  I pasted the stuff from keybase below my public key in the box on coinbase and it worked.

It should say that the text goes in the public key area with or without public key info being there also.

**Comment by AKRFranko** [comment]
@jimscarver yes! That's exactly it! Now that you have refreshed my memory! I feared the document to paste wouldn't have been a valid key. Thinking back, I should have picked up on the fact that there are START and END comments for the key, and I probably did subconsciously which is why I pasted anyways but for people who have never seen comments or code blocks before, this could be hella confusing.

**Comment by dashohoxha** [comment]
:+1: I had a moment of hesitation too, and I gave it a try by pasting the whole text.
